spec/rewrite/scanform_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
```

Comment:
issue: please rename file to `scan_form_spec` to match the two word convention. Double check other uses of the word `ScanForm` to ensure it's snake_cased or PascalCased properly